### DISCUSSION TITLE
iPadOS 15 PWA start QuickLook only in model mode and AR not working

### DISCUSSION THREAD

**Discussion by PetrSemrad** [google/model-viewer#2836] [General]
After updating to iPadOS 15 (then to 15.0.1) and installing the application in safari as PWA, after opening QuickLook it´s not possible to display the object in AR.
Only safari without installing a PWA application and opening a QuickLook object in AR works without problems
![42053](https://user-images.githubusercontent.com/91898225/135875718-2a63c3a8-9a5d-475f-99c2-4084aa0c67a4.jpg)

**Comment by ghost** [comment]
Curious, was this working on a previous version of iOS? Are you getting any error feedback that you can provide?

**Reply by PetrSemrad** [reply]
Yes, everything worked fine on the previous version of iPadOS 14.x.
It's interesting when I try as ios-src for example https://xxx.com/xxx.usdz#applePayButtonType=buy&checkoutTitle=Biplane%20Toy&checkoutSubtitle=Rustic%20finish%20with%20rotating%20propeller&price=$15
So in a safari without installing PWA, everything works, but not in the installed application. As if the safari didn't pass attributes after #

**Comment by PetrSemrad** [comment]
It looks like a bug in iPadOS
https://developer.apple.com/forums/thread/690722
